tests/integration/conftest.py

```diff
@@ -133,12 +133,17 @@ async def app_fixture(
     Builds the charm and deploys it and the relations it depends on.
     """
     # Deploy relations to speed up overall execution
-    related_apps = await asyncio.gather(
-        model.deploy("postgresql-k8s", channel="14/edge", series="jammy", trust=True),
-        model.deploy("redis-k8s", series="jammy", channel="latest/edge"),
-        model.deploy("nginx-ingress-integrator", series="focal", trust=True),
+
+    postgres_app = await model.deploy(
```

Comment:
Can we add a comment here about why we're using beta, and that we plan to switch back to edge when possible?

Reply:
Comment added